pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/HashJoinOperator.java

```diff
@@ -218,7 +218,8 @@ private void buildBroadcastHashTable()
         _resourceLimitExceededException =
             new ProcessingException(QueryException.SERVER_RESOURCE_LIMIT_EXCEEDED_ERROR_CODE);
         _resourceLimitExceededException.setMessage(
-            "Cannot build in memory hash table for join operator, reach number of rows limit: " + _maxRowsInHashTable);
+            "Exception occurred when building in-memory hash table for join operator, reach number of rows limit: "
```

Comment:
(minor) I wouldn't count this as exception, I personally prefer the existing one